### DISCUSSION TITLE
Table issue???

### DISCUSSION THREAD

**Discussion by vitali-duckma** [ant-design/ant-design#39440] [Q&A]
react_devtools_backend.js:4012 Warning: Table: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.     at Table (webpack-internal:///./node_modules/rc-table/es/Table.js:133:25)

My code:  `<Table columns={tableColumns} dataSource={data} rowKey="id"/>`

**Comment by zombieJ** [comment]
@li-jia-nan remove part of them, seems need continue handle this.

**Reply by vitali-duckma** [reply]
I'm using nextjs 13 and simple tables of antd latest version

**Reply by vitali-duckma** [reply]
![Screenshot 2022-12-09 alle 11 19 31](https://user-images.githubusercontent.com/118166188/206679939-d59f6d52-b1bb-41f9-9410-63f4363c8e6e.png)

**Comment by vfritzon** [comment]
This is still an issue in latest stable 4.x release of antd which uses rc-table@7.26.0. rc-table has fixed it in https://github.com/react-component/table/pull/913 but using the latest version of rc-table (minor version upgrade) with 4.x version of antd causes the following error:
```
   node_modules/antd/es/table/Table.js:8:9:
      8 │ import { INTERNAL_HOOKS } from "rc-table/es/Table";
```

Resolving to version `7.28.3` seems to be a viable workaround.
